modules/face/data/mean_shape.txt

```diff
@@ -0,0 +1,194 @@
+68.0391,98.5278
```

Comment:
Maybe it is good to have a first line of comment like

```
%% Mean Shape landkmark location
%% Average face dimensions
200, 200
%% Landmark locations
positions like now
```

This will help to give people an idea based on what dimensions the locations were generated. Maybe add a link even to the correct functionality for training this?